SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialLoadBalanceEnforcer.cpp

```diff
@@ -45,7 +46,12 @@ void SpatialLoadBalanceEnforcer::AuthorityChanged(const Worker_AuthorityChangeOp
 			return;
 		}
 
-		const PhysicalWorkerName* OwningWorkerId = VirtualWorkerTranslator->GetPhysicalWorkerForVirtualWorker(AuthorityIntentComponent->VirtualWorkerId);
+		if (!VirtualWorkerTranslator.IsValid())
```

Comment:
With the check for IsValid() in the constructor, the only way this could not be valid would be during destruction, when this method couldn't be called, so I think it's safe here (and below) to just check rather than testing.